src/app/containers/ATIAnalytics/README.md

```diff
@@ -2,56 +2,130 @@
 
 This directory contains code for logging events to ATI Analytics.
 
-## Log click events to ATI
+## Logging Click Events
+For logging click events you will need four things:
+* [useClickTracker](https://github.com/bbc/simorgh/blob/latest/src/app/contexts/EventContext/index.jsx) used to track a component to be tracked. This should be done by using `data-attributes` used in the component. This data attribute should also contain value for building the component info object, e.g. `data-navigation=‘Akuko_0’`
+* to compose the tracking URL via [buildATIClickParams](https://github.com/bbc/simorgh/blob/latest/src/app/containers/ATIAnalytics/params/index.js#L41) which uses [RequestContext](https://github.com/bbc/simorgh/blob/latest/src/app/contexts/RequestContext/index.jsx) and [ServiceContext](https://github.com/bbc/simorgh/blob/latest/src/app/contexts/ServiceContext/index.jsx).
+* to get all the component information by using [getComponentInfo](https://github.com/bbc/simorgh/blob/latest/src/app/lib/analyticsUtils/index.js)
+* [sendEventBeacon](https://github.com/bbc/simorgh/blob/latest/src/app/containers/ATIAnalytics/beacon/index.js) for sending the two events - ‘click’ and ‘view’.
 
-For this, you will need to 
+## How to set up tracking for a component
+In the container for the component you’re trying to track,  import `EventContext`, and use `useClickTracker` as such:
 
-- build an ATI Url, using [buildATIClickParams](./params/index.js), which depends on the [RequestContext](../../contexts/RequestContext/index.jsx) and [ServiceContext](../../contexts/ServiceContext/index.jsx).
+```
+import React, { useContext } from ‘react’;
```

Comment:
There are a few places in this file where there's a ‘ rather than a '. Text editors 🙄
```suggestion
import React, { useContext } from 'react';
```